src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_Block.vb

```diff
@@ -42,21 +42,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
```

Comment:
Methods with static locals were being half-instrumented, but without ever generating the necessary prologue (because the test for the method body block always failed). I'll correct this and add a test.